index.js

```diff
@@ -4,7 +4,7 @@
 // Mousemove scroll left/right
 
 // COLOR PALLETE AND ASSIGNMENT
-let colors = {
+const SUPPORTED_COLORS = {
```

Comment:
The Object assigned to this variable does not change, so it's a `const`.

Additionally, because the VALUES in the Object aren't being modified, it is conventional to UPPERCASE_THEM